models/exposure.py

```diff
@@ -300,6 +300,17 @@ def filter_short(self):
             'Bad exposures are each bad in their own way (i.e., have different bits set). '
     )
 
+    preproc_bitflag = sa.Column(
+        sa.SMALLINT,
+        nullable=False,
+        server_default=sa.sql.elements.TextClause( '0' ),
```

Comment:
Why is this better than simply `server_default=0`?

Reply:
....I don't remember exactly why I fell back on that.  But, when I was doing the database refactor, I figured out that just using a 0 did something strange database side that was messy there-- it had casts, or some such, that were gratuitous.  (Surprise: sqlalchemy does things that end up being clumsier database side!)  All of this is just a way of saying, "no, really, I just want to tell the database that the default is 0".)